.eslintrc

```diff
@@ -0,0 +1,15 @@
+---
+  extends: "@dosomething/eslint-config"
```

Comment:
I recommend using new version available here:
https://github.com/DoSomething/blink/blob/master/.eslintrc.yml

Note: it might be to good idea to move this to its own package specific to backend apps.